src/patternfly/components/Scrollspy/scrollspy.scss

```diff
@@ -0,0 +1,168 @@
+  --pf-c-scrollspy__link--before--BorderRightWidth: 0;
+  --pf-c-scrollspy__link--before--BorderBottomWidth: 0;
+  --pf-c-scrollspy__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--before--BorderColor: transparent;
+  --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__link--focus--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--focus--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__item--m-current__link--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+
+  // text
+  --pf-c-scrollspy__item-text--Color: var(--pf-global--Color--200);
+  --pf-c-scrollspy__link--hover__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__link--focus__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__item--m-current__item-text--Color: var(--pf-global--Color--100);
+
+  // label
+  --pf-c-scrollspy__label--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__label--PaddingBottom: var(--pf-global--spacer--md);
+
+  padding-right: var(--pf-c-scrollspy--PaddingRight);
+  padding-left: var(--pf-c-scrollspy--PaddingLeft);
+  background-color: var(--pf-c-scrollspy--BackgroundColor);
+
+  &.pf-m-background {
+    --pf-c-scrollspy--BackgroundColor: var(--pf-c-scrollspy--m-background--BackgroundColor);
+  }
+
+  &.pf-m-center {
+    text-align: center;
+  }
+
+  &.pf-m-vertical {
+    --pf-c-scrollspy__list--PaddingRight: 0;
+    --pf-c-scrollspy__list--PaddingLeft: 0;
+    --pf-c-scrollspy__list--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__list--before--BorderTopWidth);
+    --pf-c-scrollspy__list--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__list--before--BorderLeftWidth);
+    --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth);
+    --pf-c-scrollspy__link--focus--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth);
+    --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth);
+    --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth);
+
+    .pf-c-scrollspy__list {
+      flex-direction: column;
```

Comment:
you could make this a var, too

Reply:
this wasnt working for me when i tried making them vars - ill revisit if i have time